hibernate-core/src/main/java/org/hibernate/metamodel/mapping/internal/EntityVersionMappingImpl.java

```diff
@@ -87,7 +84,7 @@ public EntityVersionMappingImpl(
 
 		this.versionBasicType = versionBasicType;
 
-		unsavedValueStrategy = UnsavedValueFactory.getUnsavedVersionValue(
+		this.unsavedValueStrategy = UnsavedValueFactory.getUnsavedVersionValue(
```

Comment:
This was actually already consistent with, say `BasicEntityIdentifierMappingImpl`. I'm not a fan of redundant `this`s.

Reply:
Should I keep `this` or remove all of it?

Reply:
My rule is: use `this` only when it's necessary to disambiguate. Here it's not. Remember thy IntelliJ highlights instance variables in purple anyway, so I always know that this is setting an instvar.

Reply:
OK, I reverted this part.